### PR TITLE
Update items subscribed to 'playpause' only if there has been a change

### PR DESCRIPTION
**Pull request by morphZ**
Hi E3EAT,

I tried to propose the latest suggestion in my comments on your PR as new PR on your PR. Following this [manual](https://github.com/TeamPorcupine/ProjectPorcupine/wiki/How-to-Test-a-Pull-Request). Could you try to merge that into your pull request on the main repo?

Greetings
morphZ

### CODE REVIEW

File: mpd/__init__.py [hunk 1]
```diff
@@ -231,7 +231,9 @@ def refreshItems(self,subscribedItems,response,warn):
             #update subscribed items which do not exist in the response from MPD
             elif key == 'playpause':
                 item = subscribedItems[key]
-                self.setItemValue(item,self._internal_tems['isPlaying'])
+                val = self._internal_tems['isPlaying']
+                if item() != val:
+                    self.setItemValue(item,val)
```
Comment (E3EAT) [inline mpd/__init__.py]: this change is OK for me but please also add this change for mute and Artist. Just add another commit to this PR.

Comment (morphZ) [reply mpd/__init__.py]: done

**Comment by morphZ** [comment]
done

**Comment by E3EAT** [comment]
OK, just merged your changes. Thank you for your support so far.

**Pull request merged**